open-metadata-implementation/common-services/generic-handlers/src/main/java/org/odpi/openmetadata/commonservices/generichandlers/SchemaAttributeHandler.java

```diff
@@ -302,6 +302,15 @@ public String createNestedSchemaAttribute(String                 userId,
             {
                 schemaAttributeBuilder.setAnchors(userId, anchorGUID, methodName);
             }
+            SchemaTypeBuilder schemaTypeBuilder = schemaAttributeBuilder.getSchemaTypeBuilder();
+
+            /*
+             * if there is a a formula then set it into the schemaAttributeBuilder
+             */
+            if (schemaTypeBuilder != null && schemaTypeBuilder.isDerived())
```

Comment:
How do you clear the formula?  if isMergeUpdate == false then a null formula should mean the formula value is set to null.

Reply:
this code is in create so we do not need to handle the merge semantics.